.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Tests
+on:
+  push:
+    branchs:
+      - master
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
```

Comment:
It looks like `nose` is not supported in Python3.9+. Which is super annoying. That's going to cause me headaches soon. However, what it means is we should be able to switch this from `3.10` to `3.8` and I think it will get further along.

Reply:
https://github.com/nose-devs/nose/issues/1099

Reply:
Try switching this line to `3.8`. That should result in the GH action using 3.8.